### PR TITLE
feat: support custom cypress config file

### PR DESCRIPTION
**Pull request by tamasmagyar**
Would like to add support for custom named cypress config files.

Proposal:

- Custom cypress config file can be passed via CYPRESS_CONFIG_FILE env var

Testing:
- Tried to add the corresponding ava test, but based on my experience it is difficult to test env variables with ava
- Added demo task to the CI file

Todo:
- Upon green light for the feature, I will extend add to the documentation

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "demo-names": "DEBUG=find-cypress-specs node ./bin/find --names",
     "demo-skipped-tests": "DEBUG=find-cypress-specs node ./bin/find --names --skipped",
     "demo-count-skipped-tests": "DEBUG=find-cypress-specs node ./bin/find --names --skipped --count",
+    "demo-custom-cypress-config": "DEBUG=find-cypress-specs CYPRESS_CONFIG_FILE=test-ts/cypress.config.custom.ts node ./bin/find",
```
Comment (bahmutov) [inline package.json:21]: why not add another CLI argument?

Comment (tamasmagyar) [reply package.json:21]: just added

Comment (tamasmagyar) [reply package.json:21]: or which one do you mean?

**Review by bahmutov — APPROVED** [review]
looks good, thanks for adding a test

**Pull request merged**